test/testdata

```diff
@@ -0,0 +1,8 @@
+1 2
+2 3
+3 4
+1 3
+1 5
+1 6
+1 7
+1 8
```

Comment:
I think we do have some ordering guarantees on most of our input files, but this format matches.

Reply:
does this implicate some redundant checks in the file processing logic ?

Reply:
I don't think so. It is just a note in case we eventually see a way to take advantage of the ordering (aside from free cache benefits while loading); I currently do not.